arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -232,5 +232,17 @@ public async Task GetCollectionsAsync_ShouldSucceed()
             Assert.Equal(HttpStatusCode.OK, response.Code);
             Assert.NotNull(collectionExists);
         }
+
+        [Fact]
+        public async Task GetCollectionRevisionAsync_ShouldSucceed()
```

Comment:
Maybe also add a test for a collection name that does not exist? We should throw if arangodb returns an error in that case.